app/views/evaluation_forms/_form.html.erb

```diff
@@ -19,57 +8,93 @@
         provide instructions that will be shared with your evaluators. 
       </p>
       <div>
-        <%= form.label :title, style: "display: block" %>
-        <%= form.text_field :title %>
+        <%= form.label :title, class: "usa-label" %>
+        <%= form.text_field :title, class: "usa-input", data: {"action": "evaluation-form#validatePresence focusout->evaluation-form#validatePresence"} %>
+        <for>
       </div>
 
-      <div>
-        <%= form.label :challenge_phase, style: "display: block" %>
-        <%= form.number_field :challenge_phase %>
-      </div>
+      <%= inline_error(evaluation_form, :title) %>
 
-      <div>
-        <%= form.label :challenge_id, style: "display: block" %>
-        <%= form.text_field :challenge_id %>
+      <label class="usa-label" for="challenge-select">Select a challenge</label>
+      <div class="usa-hint">Choose the challenge this form will evaluate.</div>
+      <div class="usa-combo-box">
+        <select class="usa-select" name="challenge-select" id="challenge-select" data-action="evaluation-form#handleChallengeSelect">
+          <% current_user.challenge_manager_challenges.each do |challenge| %>
+            <% if challenge.is_multi_phase %>
+              <% challenge.phases.each.with_index(1) do |phase, index| %>
+                <option value="<%= "#{challenge.id}.#{index}.#{phase.end_date.strftime("%m/%d/%Y")}" %>"><%= challenge.title %> - Phase <%= index %></option>              
```

Comment:
giving this more thought, I want this PR to use the phase_id instead of the index of the phase in the list. this will require a migration that:
1. truncates the `evaluation_forms` table:
```
ActiveRecord::Base.connection.truncate(EvaluationForm.table_name)
```
2. drops the `phase` column
3. adds the `phase_id` column as a foreign_key on the `phases` table
4. adds `belongs_to :phase` to the EvaluationForm.